examples/MacOS-Example/MacOS-Example/Services/BluetoothService.swift

```diff
@@ -115,10 +115,12 @@ extension BluetoothService: BlueFalconDelegate {
     func didRssiUpdate(bluetoothPeripheral: BluetoothPeripheral) {
     }
 
-    func didDiscoverDevice(bluetoothPeripheral: BluetoothPeripheral) {
+    func didDiscoverDevice(bluetoothPeripheral: BluetoothPeripheral, advertisementData: [AdvertisementDataRetrievalKeys : Any]) {
         guard (devices.first {
             $0.bluetoothDevice.identifier == bluetoothPeripheral.bluetoothDevice.identifier
         } == nil) else { return }
+        print("Device advertised data: \(advertisementData)")
```

Comment:
minor: use println instead of just print

Reply:
This is swift. There is no Println. This is done automatically

Reply:
😂 sorry brain still in holiday mode.